src/main/java/com/coral/epidemicsimapiserver/configuration/RabbitMQConfig.java

```diff
@@ -5,18 +5,16 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
-
 @Configuration
 public class RabbitMQConfig {
-    public static final String GAME_UPDATE_QUEUE_NAME = "game-metrics-1";
-    private static final String GAME_UPDATE_EXCHANGE_NAME = "game-metrics";
+    public static final String GAME_UPDATE_QUEUE_NAME = "notification-queue";
```

Comment:
Should the variable (constant) be renamed to match the change of function?

Reply:
yeah, actually, i think that would make things more consistent and readable

Reply:
i can go ahead and do that